ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1065,9 +1250,24 @@ export const RuntimePanel = fp.flow(
               </FlexRow>
             </Fragment>
       ],
-      [PanelContent.Delete, () => <ConfirmDelete
+      [PanelContent.Delete, () => (enablePD && selectedCompute === ComputeType.Standard && !runtimeExists && pdExists) ?
```

Comment:
I dislike the double ternary, though I understand there are already several in this file. I would suggest just turning this into a normal function:

```
() => {
  if (enablePD && selectedCompute === ComputeType.Standard && pdExists) {
    if (runtimeExists) {
      return <ConfirmDeleteRuntimeWithPD>;
    } else {
      return <ConfirmDeleteUnattachedPD>;
    }
    return <ConfirmDelete>..;
  }
}
```